### PR TITLE
RHV-45567: Modularize the encryption assembly

### PR DESCRIPTION
**Pull request by apinnick**
Bug fix

Fixes issue #[RHV-45567](https://issues.redhat.com/browse/RHV-45567)

Changes proposed in this pull request:

- Create proper modules for encryption assembly.

Preview build: http://file.tlv.redhat.com/apinnick/RHV-45567-fix-appendix/administration_guide/index.html#appe-Red_Hat_Enterprise_Virtualization_and_SSL

I confirm that this pull request was submitted according to the [contribution guidelines](https://github.com/oVirt/ovirt-site/blob/master/CONTRIBUTING.md): @apinnick 

This pull request needs review by: @sandrobonazzola

### CODE REVIEW

File: source/documentation/administration_guide/topics/troubleshooting-vnc-sasl-ansible-playbook.adoc [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+[source,terminal,subs="normal"]
+----
+#host_key_checking = False
+----
+
+* Disable host key checking temporarily by running the following command:
++
+[source,terminal,subs="normal"]
+----
+export ANSIBLE_HOST_KEY_CHECKING=False.
```
Comment (apinnick) [inline source/documentation/administration_guide/topics/troubleshooting-vnc-sasl-ansible-playbook.adoc:25]: @sandrobonazzola 
The period at the end of the command looks odd. Should it be removed?

File: source/documentation/administration_guide/topics/running_vnc_sasl_ansible_playbook_for_each_host.adoc [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+----
+
+* If the Manager is running as a self-hosted engine, use menu:Compute[Virtual Machines] to select the self-hosted engine virtual machine, named `HostedEngine` by default, and click btn:[Console].
+
+. On the Manager machine, run the VNC SASL ansible playbook for each host.
++
+[source,terminal,subs="normal"]
+----
+# cd /usr/share/ovirt-engine/ansible-runner-service-project/project/
+# ansible-playbook --ask-pass --inventory=<hostname>, ovirt-vnc-sasl.yml
```
Comment (apinnick) [inline source/documentation/administration_guide/topics/running_vnc_sasl_ansible_playbook_for_each_host.adoc:29]: @sandrobonazzola 
Should there be a comma in this command?

Comment (sandrobonazzola) [reply source/documentation/administration_guide/topics/running_vnc_sasl_ansible_playbook_for_each_host.adoc:32]: I don't think so. @mnecas ?

File: source/documentation/administration_guide/topics/troubleshooting-vnc-sasl-ansible-playbook.adoc (hunk 1 above)
Comment (sandrobonazzola) [reply source/documentation/administration_guide/topics/troubleshooting-vnc-sasl-ansible-playbook.adoc:25]: yes, please remove

**Review by sandrobonazzola — APPROVED** [review]
LGTM

**Pull request merged**